addons/mail/data/mail_templates_email_layouts.xml

```diff
@@ -149,10 +149,6 @@
                     </t>
                     <t t-else="">
                         <span style="font-size: 10px;">Your <t t-esc="model_description or 'document'"/></span>
-                        <br/>
-                        <span style="font-size: 20px; font-weight: bold;">
-                            <t t-esc="message.record_name and message.record_name.replace('/','-') or ''"/>
```

Comment:
We don't need this anymore?
Not sure it's clear from commit message.

Reply:
Not sure actually.